src/stylesheets/abstracts/_mixins.scss

```diff
@@ -31,3 +60,17 @@
     margin-bottom: 40px;
   }
 }
+
+
+//FACTORS
+@mixin item {
+  position: absolute;
+  background-color: $color-white;
+  border-radius: 5px;
+  margin: 2.5rem 0.5rem;
+  padding: 3rem 2rem 2rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  top: -12rem;
+}
```

Comment:
Mixins are cool, but using them too much can sometimes obfuscate styles and where they're coming from. If you came back to this project in a year and saw:

```
.item {
  @include item()
}
```

Would you remember where to find this mixin?

How much do you find yourself re-typing this code? Are you re-writing these lines enough that this mixin is saving you time?

Reply:
Also.... does this need to be flex??

Reply:
resolved in pr#8